src/llms-txt/infrastructure/markdown-services/content-types-collector.php

```diff
@@ -145,9 +156,55 @@ private function get_recent_cornerstone_content( string $post_type, int $limit )
 	 * @param string $post_type The post type.
 	 * @param int    $limit     The maximum number of posts to return.
 	 *
-	 * @return array<int, array<WP_Post>> The most recently modified posts.
+	 * @return array<WP_Post> The most recently modified posts.
 	 */
 	private function get_recent_posts( string $post_type, int $limit ): array {
+		$exclude_old = false;
+
+		if ( $post_type === 'post' ) {
+			$exclude_old = true;
```

Comment:
Maybe rename this to exclude_older_than_one_year or something since that is what this does. I was very confused on what `old` meant in this case.

Reply:
haha fair enough, added [here](https://github.com/Yoast/wordpress-seo/pull/22327/commits/3c7dd0718074a2f53b2763c6c36ba8bea353345c).